src/App.tsx

```diff
@@ -1,203 +1,160 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
 import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { getTodos, addTodo, deleteTodo, updateTodo } from './api/todos';
+import TodoList from './components/TodoList';
+import Footer from './components/Footer';
+import Header from './components/Header';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTodo, setNewTodo] = useState('');
+  const [activeFilter, setActiveFilter] = useState('All');
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
It's better to name boolean states starting from the verb `is` / `has` etc.